cluster/ci/config/prow/config.yaml

```diff
@@ -151,6 +151,7 @@ presubmits:
   - name: test_pull_request_origin_extended_clusterup
     agent: jenkins
     always_run: false
+    run_if_changed: '^(pkg/oc/bootstrap)|^(test/extended/clusterup.sh)'
```

Comment:
`'^(pkg/oc/bootstrap|test/extended/clusterup.sh)' ?

Reply:
does it matter? I find the parens offer a nice separator.

Reply:
no